src/lib/output/HtmlOutputFormatter.ts

```diff
@@ -30,7 +30,7 @@ export class HtmlOutputFormatter implements OutputFormatter {
 						ruleName: v.ruleName,
 						category: v.category,
 						url: v.url,
-						message: this.verboseViolations && result.engine === ENGINE.RETIRE_JS ? v.message.replace(/\n/g, '<br>') : v.message, // <br> used for line breaks in html
+						message: this.verboseViolations && (result.engine as ENGINE) === ENGINE.RETIRE_JS ? v.message.replace(/\n/g, '<br>') : v.message, // <br> used for line breaks in html
```

Comment:
Updating so our self evaluation passes with the new eslint rules. Updated a number of files so that we do not compare a string with an enum. Both need to be string or both need to be enums.